src/__tests__/recurring.spec.ts

```diff
@@ -66,7 +66,7 @@ afterEach(() => {
 });
 
 describe("Recurring", (): void => {
-    test.each([false, true])("should test have recurring details list, isMock: %p", async (isMock): Promise<void> => {
+    test.each([true])("should test have recurring details list, isMock: %p", async (isMock): Promise<void> => {
```

Comment:
Here it's a shorter version for this test:
```suggestion
    test("should list recurring details ", async (): Promise<void> => {
        scope.post("/listRecurringDetails")
            .reply(200, listRecurringDetailsSuccess);
        const request = createRecurringDetailsRequest();
        
        const result = await recurring.listRecurringDetails(request);
        
        expect(result).toBeTruthy();
        expect(result.details?.[0].RecurringDetail.recurringDetailReference).toBe('recurringReference');
    });
```